src/ucs/config/parser.c

```diff
@@ -1851,6 +1852,64 @@ void ucs_config_parser_print_all_opts(FILE *stream, const char *prefix,
     }
 }
 
+static void ucs_config_parser_concat_fuzzy_matches_recurs(
+        const ucs_config_field_t *fields, const char *env_prefix,
+        const char *table_prefix, const char *user_var,
+        ucs_string_buffer_t *fuzzy_buff)
+{
+    char ucx_var[128]                 = {0};
```

Comment:
WDYT about using `ucs_string_buffer_t` instead of char array here?

Reply:
isn't it a bit overhead? 
it will make the code a little more complicated and not very nice to look at.

Reply:
also AFAIU `ucs_string_buffer_t` is allocated dynamically on the heap, which is problematic for us, because it means we will allocate buffer per comparison (for each pair of {UCX_VAR, unused_var}